### PR TITLE
fix: eth address example to weth address

### PR DESCRIPTION
**Pull request by platocrat**

### CODE REVIEW

File: src/docs/developers/integration.md [hunk 1]
```diff
@@ -125,15 +125,24 @@ Here's a step-by-step process on how to do that while in your browser:
 
 ![Custom RPC button](../../assets/custom-metamask-network-1.png)
 
-3. Next, you'll enter in the network parameters. Enter `Optimistic Ethereum (Local L2)` for the network name, `http://0.0.0.0:8545` for the RPC URL, and `420` for the chain ID -- additionally, you can also set the currency symbol to `ETH`, but that's not entirely necessary since the use of ETH is implicitly understood:
+3. Next, you'll enter in the network parameters. Enter `Optimistic Ethereum (Local L2)` for the network name, `http://0.0.0.0:8545` for the RPC URL, and `420` for the chain ID and click save.
```
Comment (maurelian) [inline src/docs/developers/integration.md:128]:
```suggestion
3. Next, you'll enter in the network parameters: 
- network name: `Optimistic Ethereum (Local L2)`
- RPC URL: `http://0.0.0.0:8545` for the RPC URL
- chain ID: `420`

Then click save.
```
Lists are more readable for things like this IMO.

File: src/docs/developers/integration.md [hunk 2]
```diff
@@ -125,15 +125,24 @@ Here's a step-by-step process on how to do that while in your browser:
...
 
 ![Entering network params](../../assets/custom-metamask-network-2.png)
 
-4. Click save! And, you're done! 🙌 😎. You'll then see something like the image below when you click on your new `Optimistic Ethereum (Local L2)` network!
+4. Next, we need to add the ETH address that is used on L2.
 
-![Saving custom network](../../assets/complete-custom-network.png)
+To do this, we need to add the **WETH address** address to our account.
```
Comment (maurelian) [inline src/docs/developers/integration.md:134]:
```suggestion
To do this, we need to add the WETH address to our account.
```

Aside: I think OVM_ETH would be a better term to use eventually but consistency is more important for now.

File: src/docs/developers/integration.md [hunk 3]
```diff
@@ -125,15 +125,24 @@ Here's a step-by-step process on how to do that while in your browser:
...
 
-Adding the currency symbol is _optional_.
+![Adding WETH address](../../assets/add-weth-to-account.png)
+
+The WETH address to add is `0x4200000000000000000000000000000000000006`.
+You should see that the "Token Symbol" is `ETH` if you entered the address correctly.
+Despite this field reading `ETH`, it's important to understand that **this is actually an ERC20 version of ETH that we call WETH.**
+
+::: warning L2 WETH address  
+It's very important to note here that the default ETH address that is provided by MetaMask is **NOT** the same as the WETH ([more info here](../protocol/evm-comparison.html#native-weth)), the default type of ETH that is used on L2, we will be using to make transfers between one L2 address and another.
```
Comment (maurelian) [inline src/docs/developers/integration.md:143]: Too many commas in this sentence, I'm really not sure how to read it... 
Is it saying that there are two contracts that represent ETH? 
Is there OVM-ETH and WETH?

**Review by maurelian — APPROVED** [review]
LGTM once my comments/suggestions are addressed.

File: src/docs/developers/integration.md (hunk 1 above)
Comment (platocrat) [reply src/docs/developers/integration.md:128]: Agreed!

**Pull request merged**